pdsfile/pds4file/__init__.py

```
@@ -149,6 +153,7 @@ def set_easylogger(cls):
     # Data set-specific rules are implemented as subclasses of Pds4File
     # from pdsfile_reorg.Pds4File.rules import *
     from .rules import (cassini_iss,
+                        cassini_uvis_solarocc_beckerjarmak2023,  # noqa: F401
                         cassini_vims,
```

Ruff (0.12.2)

`.rules.cassini_vims` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)